2-mandatory/2-financial-times.js

```diff
@@ -5,6 +5,13 @@
     Implement the function below, which will return a new array containing only article titles which will fit.
 */
```

Comment:
This look good!
For extra practice, can you try re-writing this using the `filter` array method?